### PR TITLE
Update contributing guide with steps to add an api endpoint

### LINKED ISSUE #46: Update contributing guide with steps to add an api endpoint
1. How to add an endpoint
2. How our folders are structured
3. How to add tests
4. How to add OpenAPI comments

### PR DESCRIPTION
**Pull request by bpvcode**
## Describe what to do

- [X] How to add an endpoint
- [X] How our folders are structured
- [X] How to add tests - Replace information from README.md to CONTRIBUTING.md
- [X] How to add OpenAPI comments
- [X] Change the commit messages section from README.md to CONTRIBUTING.md

## Issue to resolve
#46

### CODE REVIEW

**Comment by bpvcode** [comment]
Hello @ageddesi 

I am late on this, but already working on issue #46 as we talk :)
This is just a draft for now, but I am planning to have this done by the end of the day :)
Can you please assign this PR to me? Probably because I'm not a maintainer I can't assign an assignee or a reviewer.

Thank you :)

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -283,3 +283,93 @@ The file structures in this repository should look like this :
...
+        });
+    });
+}
+```
+
+Note: This endpoint will fetch all mocked users stored in `./modules/users/data/users.ts`
+
+### How our folders are structured
+
+    Mocked-API
```
Comment (ageddesi) [inline CONTRIBUTING.md:342]: We have already just done a new pass on what the folder structure will be and the naming conventions and has recently been added to this file on dev. 👍 If you could point to that would be ace.

Comment (ageddesi) [reply CONTRIBUTING.md:342]: Can you also if you are able move the commit message standards from the README.md file into this file.

Comment (bpvcode) [reply CONTRIBUTING.md:342]: Ok, i will :)

Comment (ageddesi) [reply CONTRIBUTING.md:342]: Thank you. Great work

Comment (bpvcode) [reply CONTRIBUTING.md]: Thanks @ageddesi  ;)
One more question.
Should i reference "How to add tests" to the "Testing" section of `README.md` file as well?

A screenshot of the `README.md` file

image

Comment (ageddesi) [reply CONTRIBUTING.md]: Hi @bpvcode best to move testing into the contributing guide as well.

**Comment by bpvcode** [comment]
Hello @ageddesi ,

Everything is done now :)

Please review and if you need something else just tell me.

Thanks :)

**Review by ageddesi — APPROVED** [review]

**Comment by ageddesi** [comment]
Hi @bpvcode Perfecto, thank you for contributing 👍 Happy Hacktoberfest

**Pull request merged**